test/stress_tests.py

```diff
@@ -174,7 +174,7 @@ def tearDown(self):
       task_reply_object = TaskReply.GetRootAsTaskReply(message, 0)
       local_scheduler_ids.append(task_reply_object.LocalSchedulerId())
 
-    self.assertEqual(len(set(local_scheduler_ids)), self.num_local_schedulers)
+    self.assertEqual(len(set(local_scheduler_ids)), self.num_local_schedulers + 1)
```

Comment:
Can you document that the +1 is here because the driver task does not have a local scheduler ID?